solutions/instances/README.md

```diff
@@ -1,16 +1,16 @@
-# IBM Observability Instances Deployable Architecture
+# IBM Cloud observability instances deployable architecture
 
-This architecture creates the observability instances on IBM Cloud and supports provisioning the following resources:
+This deployable architecture creates observability instances in IBM Cloud and supports provisioning the following resources:
 
-- A resource group, if one is not passed in.
-- A Log Analysis instance.
-- A Cloud Monitoring instance.
-- Creates a Cloud Object Storage (COS) instance or supports using an existing one.
-- Supports creating KMS root keys in an existing KMS instance or using existing keys if creating new buckets.
-- Creates a KMS encrypted COS bucket that is required to store archived logs or using an existing bucket.
-- Creates a KMS encrypted COS bucket for setting up Activity Tracker event routing or using an existing bucket.
-- Creates the Activity Tracker event route to a COS target.
+* A resource group, if one is not already passed in.
+* A Log Analysis instance.
+* A Cloud Monitoring instance.
+* A Cloud Object Storage instance, if one exists, or creates an instance.
+* A key management service (KMS) instance that contains KMS root keys when buckets are created, or creates KMS root keys in an existing KMS instance.
```

Comment:
It requires update. Details are:
It does not provision(create) KMS instance. KMS instance has to be existing. This deployable architecture can create keys in KMS instance or use the existing ones. Keys will be used if DA creates COS buckets.

Reply:
Does this updated wording capture the correct meaning?

"A key management service (KMS) instance that contains KMS root keys when buckets are created, if a KMS instance does not exist."

Reply:
Not exactly. How about the following:
"Creates root keys in an existing Key Management Service (KMS) if the keys do not exist. The keys are used if Cloud Object Storage buckets are created."

Reply:
We need to use a parallel structure to the other items in this bulleted list. How about this:

"An existing key management service (KMS) in which root keys are created, if the keys do not exist. These keys are used when Cloud Object Storage buckets are created."